test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1

```diff
@@ -63,12 +63,17 @@ Describe "Test-Connection" -tags "CI" {
 
```

Comment:
@JamesWTruher, your last commit had 3 failures in `PowerShell-CI-windows`
DSC MOF Compilation.Should be able to compile a MOF from a basic configuration
```powershell
Exception calling "Create" with "1" argument(s): "At line:5 char:17
+                 nxFile f1
+                 ~~~~~~
Undefined DSC resource 'nxFile'. Use Import-DSCResource to import the resource."
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\PSDesiredStateConfiguration\MOF-Compilation.Tests.ps1: line 29
29:         [Scriptblock]::Create(@"

```
DSC MOF Compilation.Should be able to compile a MOF from another basic configuration
```powershell
Exception calling "Create" with "1" argument(s): "At line:10 char:28
+                     User = "root";
+                            ~~~~~~
Unexpected token '"root"' in expression or statement.

At line:5 char:17
+                 nxScript f1
+                 ~~~~~~~~
Undefined DSC resource 'nxScript'. Use Import-DSCResource to import the resource."
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\PSDesiredStateConfiguration\MOF-Compilation.Tests.ps1: line 48
48:         [Scriptblock]::Create(@"

```
DSC MOF Compilation.Should be able to compile a MOF from a complex configuration
```powershell
Expected no exception to be thrown, but an exception "Errors occurred while processing configuration 'WordPressServer'." was thrown from D:\a\1\s\src\powershell-win-core\bin\release\netcoreapp3.1\win7-x64\publish\Modules\PSDesiredStateConfiguration\PSDesiredStateConfiguration.psm1:3745 char:5
    +     throw $ErrorRecord
    +     ~~~~~~~~~~~~~~~~~~.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\PSDesiredStateConfiguration\MOF-Compilation.Tests.ps1: line 173
173: "@) | Should -Not -Throw
```